.github/workflows/test-ipynb.yml

```diff
@@ -0,0 +1,42 @@
+name: Testing Jupyter Notebooks
+
+on:
+  push:
+    branches:
+      - "**"
+  pull_request:
+    branches:
+      - "**"
+
+jobs:
+  test:
+    timeout-minutes: 30
+
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
+        python-version: ['3.10', '3.11', '3.12']
```

Comment:
I think we should concentrate on a single python version (3.12?) but include more OS-es. This is especially important for assuring file paths are properly set on Windows/Linux

Reply:
Then let's have a full matrix with 3 python versions and 3 os versions to make sure everything is ok for all major combinations. Added to workflow config.